bin/spryker-sdk.sh

```diff
@@ -1,5 +1,10 @@
 #!/bin/bash
 
+if ! docker info > /dev/null 2>&1; then
```

Comment:
I'm not sure it actually works. I've tried docker info for the started and stopped docker service. And the result is the same.